src/ui/widgets/InputTraceWidget/SteeringWheel/SteeringWheel.module.scss

```
@@ -153,3 +153,70 @@ $groove-px: 8;
+   The outline sits at the secondary text step — at full contrast it would make
+   the wheel the brightest thing in the widget, which it is not.
+   The stroke is device-width rather than viewBox-width, so wheels traced from
+   photos of different resolutions all draw the same line. */
+.art {
+  width: 100%;
+  height: 100%;
+  color: $widget-text-secondary;
+
+  path {
+    fill: rgba(22, 22, 28, 0.92);
+    stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)